trymito.io/components/CTAButtons/CTAButtons.tsx

```diff
@@ -9,7 +9,7 @@ const CTAButtons = (props: {variant: 'download' | 'contact'}): JSX.Element => {
         <div className={styles.cta_buttons_container}> 
             {props.variant === 'download' && 
                 <TextButton 
-                    text='Install Mito for Free'
+                    text='pip install mitosheet'
```

Comment:
Let's make this copyable also or move back to the previous text. I think moving back to previous text is probably better because not all of our audience is going to know what `pip install` as a cat is